src/errors/BadRequestError.js

```diff
@@ -1,9 +1,23 @@
 // Import base error
 const { BaseIsomerError } = require("@errors/BaseError")
 
+const { ComponentTypes, FileCodes } = require("./IsomerError")
+
 class BadRequestError extends BaseIsomerError {
-  constructor(message) {
-    super(400, message)
+  constructor(
+    message,
+    meta = {},
+    componentCode = ComponentTypes.Other,
+    fileCode = FileCodes.Undefined
```

Comment:
as mentioned earlier, will probably shift these into a separate interface and get `BadRequestError` to implement that

Reply:
yes agreed